IdentityCore/src/cache/metadata/MSIDAccountMetadataCacheAccessor.m

```diff
@@ -147,7 +147,7 @@ - (MSIDAccountMetadataState)signInStateForHomeAccountId:(NSString *)homeAccountI
     }
     
     NSError *localError;
-    MSIDAccountMetadataCacheItem *cacheItem = [self retrieveAccountMetadataCacheItemForClientId:clientId skipCache:self.skipMemoryCacheForAccountMetadata context:context error:&localError];
+    MSIDAccountMetadataCacheItem *cacheItem = [self retrieveAccountMetadataCacheItemForClientId:clientId skipCache:YES context:context error:&localError];
```

Comment:
should the skipMemoryCacheForAccountMetadata  be removed now too if it doesn't do anything?

Reply:
We still use in `updatePrincipalAccountIdForClientId:` method.